composer.json

```diff
@@ -20,17 +20,19 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
         "symfony/framework-bundle": "~3.4|~4.0|~5.0",
         "doctrine/doctrine-bundle": "~1.0|~2.0",
-        "doctrine/migrations": "^2.2"
+        "doctrine/migrations": "^3.0@dev"
```

Comment:
Adding a comment here so that we don't forget to change this before merging

Reply:
We can also leave this here while this is 3.0.x-dev and update it once we enter the beta phase.